### ISSUE TITLE
Update HelloGOV Alt Text on Events Page: our-locations-content.html

### ISSUE THREAD

**Issue by Sparky-code** [hackforla/website#3281]
### Overview

As developers we want to ensure that we ensure that alt text for images adhere to Web Content Accessibility Guidelines (WCAG). This issue addresses the alt text for the HelloGOV event.

### Action Items

- [ ] In the file _includes/events-page/our-locations-content.html, change the following alt text:

From
```
HelloGOV Project
```
To
```
alt=""
```

- [ ] **For merge team**: Once the pull request associated with this issue is approved and merged, please update and edit epic #3132 by
  - [ ] Checking off the dependency for this issue
  - [ ] If all dependencies are checked off, please move issue #3132 to the New Issue Approval column and remove the Dependency label

### Resources/Instructions
- This issue is part of epic #3132 
- [Events page](https://www.hackforla.org/events/)
- [Alt text decision Tree ](https://www.w3.org/WAI/tutorials/images/decision-tree/)
- [FAE rule 4: alt text should be less than 100 characters](https://fae.disability.illinois.edu/rulesets/IMAGE_4_EN/)
- Working on an Issue: 
https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#24-claiming-an-issue

#### File and code links you will need to work on this issue
- Directory to find the page once you have it in your IDE: `_includes/events-page/our-locations-content.html`
- Click on the link below to see the source file code:
https://github.com/hackforla/website/blob/c88d715eeaca83a1665cfce48be1bead3f203fde/_includes/events-page/our-locations-content.html#L5

#### Information about WCAG
- [WCAG](https://www.w3.org/WAI/standards-guidelines/wcag/)
- [Web Accessibility Tutorials: Decorative Images](https://www.w3.org/WAI/tutorials/images/decorative/)
- [WCAG section on alt text](https://www.w3.org/WAI/WCAG21/Techniques/html/H37.html)

### Is there a dependency?

No

### If Yes, please explain

N/A

**Comment by JessicaLucindaCheng** [comment]
- Due to changes in #3027, this issue is no longer valid because that image no longer exists on the page.
